neard/src/main.rs

```diff
@@ -243,6 +243,19 @@ fn main() {
                 near_config.client_config.archive = true;
             }
 
+            #[cfg(feature = "sandbox")]
+            {
```

Comment:
Well, this check is a bit fragile.

Reply:
:) the purpose of the check is to avoid misoperation by not malicious user, sandbox node slip into mainnet/testnet is dangerous.